serializer/serializable.go

```diff
@@ -316,7 +324,6 @@ func (l LexicalOrdered32ByteArrays) Swap(i, j int) {
 
 // RemoveDupsAndSortByLexicalOrderArrayOf32Bytes returns a new SliceOfArraysOf32Bytes sorted by lexical order and without duplicates.
 func RemoveDupsAndSortByLexicalOrderArrayOf32Bytes(slice SliceOfArraysOf32Bytes) SliceOfArraysOf32Bytes {
-
```

Comment:
Why do you remove that beautiful empty line? :(

Reply:
:joy: not me, `"editor.formatOnSave": true,`